src/main/scala/codacy/pylint/Pylint.scala

```diff
@@ -14,17 +15,24 @@ object Pylint extends Tool {
     def isEnabled(issue: Issue) =
       conf.map(_.exists(_.patternId == issue.patternId)).getOrElse(true)
 
-    commandFor(path, conf, files).flatMap { case cmd =>
-      val wrappedCmd = Seq("bash", "-c", cmd.mkString(" "))
-
-      Try(wrappedCmd.lineStream_!(ProcessLogger(_ => ()))).map { lines =>
-        lines.flatMap(parseLine).filter(isEnabled)
-      }
+    def buildFileCommands(collected_files: Map[String, Array[String]]) = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name 'collected_files' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/Codacy/codacy-pylint/file/1732719714/issues/source?bid=2829823&fileBranchId=3054541#l18)